### PR TITLE
Vipcup

### PR DESCRIPTION
**Pull request by vic85821**

### CODE REVIEW

File: src/data/datasets/vipcup_seg_dataset.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                }
+            }
+            ct, = self.transforms(ct, **transforms_kwargs)
+            ct, gt = self.to_tensor(ct, gt)
+        metadata = {'input': ct, 'target': gt}
+
+        if self.type == 'test':
+            metadata.update(affine=nii_img.affine,
+                            header=nii_img.header,
+                            name=ct_path.name.replace('volume', 'segmentation'))
```
Comment (peter850706) [inline src/data/datasets/vipcup_seg_dataset.py]: Seems need to change.

**Review by peter850706 — COMMENTED** [review]
Reviewed.

File: src/data/datasets/vipcup_seg_dataset.py (hunk 1 above)
Comment (vic85821) [reply src/data/datasets/vipcup_seg_dataset.py]: Yes, I'll fix it.

**Review by peter850706 — COMMENTED** [review]
Reviewed.

**Pull request merged**